feature-libs/product-configurator/common/components/configurator-cart-entry-info/configurator-cart-entry-info.component.html

```diff
@@ -1,24 +1,26 @@
 <ng-container *ngIf="orderEntry$ | async as orderEntry">
-  <ng-container *ngIf="hasStatus(orderEntry)">
-    <div
-      class="cx-configuration-info"
-      *ngFor="let info of orderEntry.configurationInfos"
-    >
-      <div class="cx-label">{{ info?.configurationLabel }}:</div>
-      <div class="cx-value">
-        {{ info?.configurationValue }}
+  <ng-container *ngIf="isAttributeBasedConfigurator(orderEntry)">
```

Comment:
I think, it is breaking change

Reply:
No, this does not change the DOM tree for existing configurators. Please un-resolve if I miss something!